src/osdc/Objecter.h

```diff
@@ -1144,16 +1145,16 @@ class Objecter : public md_config_obs_t, public Dispatcher {
   using Dispatcher::cct;
   std::multimap<string,string> crush_location;
 
-  atomic_t initialized;
+  atomic_bool initialized;
```

Comment:
similar to https://github.com/ceph/ceph/pull/9138#discussion_r64237097, can we qualify these with `std::`?